pkgs/applications/video/anime-downloader/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, python3Packages, aria2, mpv, nodejs, fetchFromGitHub }:
+with python3Packages;
+buildPythonApplication rec {
+  pname = "anime-downloader";
+  version = "5.0.14";
+  src = fetchFromGitHub ({
+    owner = "anime-dl";
+    repo = "anime-downloader";
+    rev = "a509e4a8054665180bb2f172b3c0883a5a7da0ce";
```

Comment:
Why are you using a Git hash here, when you should use the `version` defined above?